docs/New-Maintainer-Checklist.md

```diff
@@ -79,6 +79,8 @@ If they are elected to the Homebrew's [Project Leadership Committee](https://doc
 
 If there are problems, ask them to step down as a maintainer and revoke their access to all of the above.
 
+In interests of loosely verifying maintainer identity and building camaraderie, if you find yourself in the same town (e.g living, visiting or at a conference) as another Homebrew maintainer you should make the effort to meet up. If you do so, you can expense your meal (within SFC policies). This is a more relaxed version of e.g. the Debian system to meet in person to sign keys with a legal ID verification.
```

Comment:
To clarify, we’re talking about expensing the meals for both parties? This is quite a fun idea!

Reply:
Yup! That's what we've done at GitHub also as a gentle nudge and, with a peek at Homebrew's finances, we have enough funds to do that if people aren't being silly about it.

Reply:
@alebcay Durham Homebrew meetup? 😀

Reply:
Having neglected my GitHub notifications for about five months 😅, I've missed this, but if you're still up for it, let's meet up sometime!